### DISCUSSION TITLE
Carla is running so slow!

### DISCUSSION THREAD

**Discussion by Zeinab-Rahmani** [carla-simulator/carla#5132] [Q&A]
I build Carla both using package installation and Linux build. everything is fine except that when I run the server (either by `male launch` for Linux build or `./CarlaUE.sh` for package installation) the town doesn not apper properly and I only see a vigue scene. 

To solve this, I tried to run the server in a low quality mode using `./CarlaUE4.sh -quality-level=Low`. Now, town is shown better but after run PythonAPI/Examples, the server is so slow while I belive my system specifications meet Carla requirement:
```
CPU: i9-10900k
GPU: RTX 3070
RAM: 64GB DDR4(4*16gb)
```
1. Is it my system specifications that can not run Carla properly or something else could be wrong?

2. Can I run the server in Carla 0.9.13 Linux Build without using UE? If it is not possible to skip the UE (in Carla 0.9.13 Linux Build), how can I run the server in low quality mode as I did for package installation (`./CarlaUE4.sh -quality-level=Low`)?

Thank you!

**Comment by RobTranquillo** [comment]
Hi, how many fps on the server and client do you get? Is your graphic accelarator well supported from the OS and all drivers installed? Do you checked it with some other application? Has your CPU some kind of GPU included, can you disable it?
regards

**Comment by kslhuy** [comment]
have you find a solution ?
